apps/dashboard/src/data/editor/reducer.js

```diff
@@ -42,7 +49,32 @@ const hasUnsavedChanges = ( state = false, action ) => {
 	return state;
 };
 
+const title = ( state = '', action ) => {
+	if ( action.type === EDITOR_TITLE_SET ) {
+		return action.title;
+	}
+
+	if ( action.type === PROJECT_UPDATE ) {
```

Comment:
The editor state should only contain changes waiting to be saved. I'd remove this.
it's also confusing because now this gets updated whenever `PROJECT_UPDATE` gets called for any project at any time.

Reply:
I'm not sure how to replace this one. The goal here is to catch when the project has been loaded from API and set the title from that source... where would you put it (other than in a control)?

I think part of the issue here is that, on the editor state branch, we treat things separately. So, a CONTENT_UPDATE won't/shouldn't affect `editor.title`, yet a PROJECT_UPDATE (only triggered after save/load) should.

Reply:
For now we can just do `projectTitle || project.title || default` in the project navigation thingy for the title value.  
We might want to talk about it some more, agreed... 
Perhaps just add dedicated selectors inside editor state so we don't have to solve these manually wherever they appear? Where it takes the value from editor state or falls back to `state.projects.id` using the projectId currently in `state.editor`... This is getting so confusing so quickly, lol.